### PR TITLE
doc/user: Fixed syntax error in code example

### PR DESCRIPTION
**Pull request by parkerhendo**
Fixing a syntax issue in the code sample in section 4.d of the [Kafka integration docs](https://materialize.com/docs/integrations/confluent-cloud/#create-a-source-in-materialize)

### CODE REVIEW

**Review by benesch — APPROVED** [review]
Thank you!

File: doc/user/content/integrations/confluent-cloud.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ The process to connect Materialize to a Confluent Cloud Kafka cluster consists o
           SASL PASSWORD = SECRET confluent_password;
 
       CREATE SOURCE <topic-name>
-        FROM KAFKA CONNECTION confluent_cloud TOPIC '<topic-name>'
-        FORMAT BYTES
+        FROM KAFKA CONNECTION confluent_cloud (TOPIC '<topic-name>')
+        FORMAT BYTES;
```
Comment (bkirwi) [inline doc/user/content/integrations/confluent-cloud.md:70]:
```suggestion
        FORMAT BYTES
```

**Review by bkirwi — COMMENTED** [review]
I think this snuck in!

**Review by morsapaes — APPROVED** [review]
Thanks, @parkerhendo! 🎉

**Pull request merged**